src/main/scala/nl.knaw.dans.easy.authinfo/EasyAuthInfoApp.scala

```diff
@@ -86,7 +104,7 @@ trait EasyAuthInfoApp extends AutoCloseable with DebugEnhancedLogging with Appli
     for {
       ddm <- bagStore.loadDDM(bagId)
       ddmProfile <- getTag(ddm, "profile", bagId)
-      dcmiMetadata = getOptionalTag(ddm, "dcmiMetadata")
+      dcmiMetadata = (ddm \ "dcmiMetadata").headOption
```

Comment:
I suppose this method used to be extracted to mirror `getTag`.

Reply:
yes, but it was only used once, so it makes more sense to just have it inline for now.